my_list.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'MyEnumerable'
```

Comment:
- [ ] Please kindly import `my_enumerable.rb` file properly, this will make the app functionality accessible. 👍 

```suggestion
require_relative 'my_enumerable'
```